instrumentation/tomcat-7.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/tomcat7/TomcatTracer.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.tomcat7;
+
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.propagation.TextMapPropagator;
+import io.opentelemetry.instrumentation.api.servlet.AppServerBridge;
+import io.opentelemetry.instrumentation.api.tracer.HttpServerTracer;
+import java.net.URI;
+import java.util.Collections;
+import org.apache.coyote.Request;
+import org.apache.coyote.Response;
+import org.apache.tomcat.util.buf.MessageBytes;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class TomcatTracer extends HttpServerTracer<Request, Response, Request, Request>
+    implements TextMapPropagator.Getter<Request> {
+
+  private static final Logger log = LoggerFactory.getLogger(TomcatTracer.class);
+  private static final TomcatTracer TRACER = new TomcatTracer();
+
+  public static TomcatTracer tracer() {
+    return TRACER;
+  }
+
+  public Context startServerSpan(Request request, String adapterClassName) {
+    Context context =
+        AppServerBridge.init(startSpan(request, request, request, adapterClassName + ".service"));
+
+    // context must be reattached, because it has new attributes compared to the one returned from
+    // startSpan().
+    attachServerContext(context, request);
+    return context;
+  }
+
+  @Override
+  protected String getInstrumentationName() {
+    return "tomcat";
```

Comment:
Should be "io.opentelemetry.javaagent.tomcat" to be consistent with other integrations

Reply:
Yes, fixed.